### PR TITLE
fix logread and update ccore

### PR DESCRIPTION
**Pull request by xjlgod**

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Response:
...
+{"configBody":{"version":"v2","description":"web console import","clientSettings":{"retry":3,"concurrency":10,"channelBufferSize":128,"space":"basketballplayer","connection":{"user":"root","password":"123","address":"192.168.8.145:9669"}},"logPath":"E:\\NebulaProject\\test\\nebula-studio\\server\\data\\tasks/import.log","files":[{"path":"E:\\NebulaProject\\test\\nebula-studio\\server\\data\\upload\\player.csv","failDataPath":"E:\\NebulaProject\\test\\nebula-studio\\server\\data\\tasks/err/数据源 1Fail.csv","batchSize":10,"type":"csv","csv":{"withHeader":false,"withLabel":false},"schema":{"type":"vertex","vertex":{"vid":{"index":0,"type":"string"},"tags":[{"name":"player","props":[{"name":"name","type":"string","index":1},{"name":"age","type":"int","index":2}]}]}}}]},"configPath":""}
```
Comment (veezhang) [inline server/README.md]: Please do not use Chinese.

File: server/Makefile [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+all: tidy check build
 
 check: fmt vet
 
+tidy:
+	go mod tidy
+
 build:
-	go build -o server main.go
+	go build -o nebula-studio-server main.go
```
Comment (veezhang) [inline server/Makefile:9]: Have you discussed with @hetao92 ?

Comment (xjlgod) [reply server/Makefile:9]: yes, he said it is ok.

**Review by veezhang — APPROVED** [review]
LGTM

**Pull request merged**